recipe/meta.yaml

```diff
@@ -28,28 +28,29 @@ requirements:
     - cartopy
     - cf-units
     - cftime
+    - compilers
```

Comment:
Why is the `compilers` package needed?

Reply:
it's in our environment.yml, bud - probably not needed here (nor in the environment.yml for the same reason)